archive/work-package/values.yaml

```diff
@@ -32,6 +32,8 @@ config:
 cmd: wps run-rest
 # cmd: wps consume-events
 
+_consumer_group:
```

Comment:
As already discussed I would suggest to set a default in the base charts values.